src/components/Pluralize/Pluralize.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+interface PluralizeProps {
+  count: number;
+  singular: string;
+  plural?: string;
+  showCount?: boolean;
+}
+
+const Pluralize: React.FC<PluralizeProps> = ({
+  count,
+  singular,
+  plural,
+  showCount,
+}) => {
+  return (
+    <>
+      {showCount && <>{count} </>}
+      {count === 1 ? singular : plural || `${singular}s`}
```

Comment:
This expression took me a few seconds to parse. I'd prefer the "fallback" behavior be defined separately, so as to simplify the expression:

```diff
+ const pluralStr = plural || `${singular}s`;
  return (
    <>
      {showCount && <>{count} </>}
-     {count === 1 ? singular : plural || `${singular}s`}
+     {count === 1 ? singular : pluralStr}
``

Reply:
Also, here's a link to an [NPM package](https://github.com/plurals/pluralize) (plus TS [types](https://www.npmjs.com/package/@types/pluralize) for it) called "pluralize" that might come in handy for us in the future. I've never used it before. I think it was designed to handle some of the pluralization special cases in English.

Reply:
I took your suggestion on making the logic more clear. The extra space in `<>{count} </>` is intentional. Otherwise Reach will collapse the whitespace between the two expressions and you'd end up with "2Samples". I added a comment to explain that.